src/main/drivers/sdcard_sdio_baremetal.c

```diff
@@ -306,6 +333,7 @@ static bool sdcard_isReady()
 static sdcardOperationStatus_e sdcard_endWriteBlocks()
 {
     sdcard.multiWriteBlocksRemain = 0;
+    if (sdcard.useCache) cache_reset();
```

Comment:
Please use blocks even for single line conditional expressions. This makes code easier to maintain, and reduces the potential for bugs.